DZ_lesson1_Vaikule_Nataliia

NataliaHimera opened the pull request:
Дмитрий, здравствуйте! 
Высылаю ДЗ к уроку 1.
Благодарю.

NataliaHimera committed:
Update task_1.py

NataliaHimera committed:
Update task_2.py

NataliaHimera committed:
Update task_3.py

NataliaHimera committed:
Update task_4.py

NataliaHimera committed:
Update task_5.py

NataliaHimera committed:
Update task_6.py

DmitryChitalov commented on Урок 1. Практическое задание/task_1.py:
```diff
@@ -11,3 +11,20 @@
+age = input("How old are you?: ")
+print("Hello " + name + "!")
+print(age + " is a great age to learn Python!")
+# Но здесь возраст все же не будет числом, т. к. тип str. 
```
print("Hello " + name + "!")
print(age + " is a great age to learn Python!")
конкатенация - плохой стиль
применяйте ф-строки

NataliaHimera replied:
name = input("What is your name?: ")
age = input("How old are you?: ")
print(f"Hello {name}!")
print(f"{age} is a great age to learn Python!")

DmitryChitalov commented on Урок 1. Практическое задание/task_2.py:
```diff
@@ -9,3 +9,14 @@
+time.gmtime(0)
+r = int(input("Введите время в секундах: "))
+time_format = time.strftime("%H:%M:%S", time.gmtime(r))
+print("Время: ", time_format)
```
выполнено

DmitryChitalov commented on Урок 1. Практическое задание/task_3.py:
```diff
@@ -8,3 +8,13 @@
+d = n * 11
+i = n * 111
+f = n + d + i
+print(f)
```
выполнено

DmitryChitalov commented on Урок 1. Практическое задание/task_4.py:
```diff
@@ -10,3 +10,17 @@
+    g //= 10
+    if j > h:
+        h = j
+print(h)
```
выполнено

DmitryChitalov commented on Урок 1. Практическое задание/task_5.py:
```diff
@@ -18,3 +18,20 @@
+    print("Ваша прибыль составила " + str(k) + " руб.")
+else:
+    print("К сожалению, ваша фирма терпит убытки((")
+    print("Ваш убыток составил " + str(k) + " руб.")
```
нет elif

DmitryChitalov replied:
а если равны?

DmitryChitalov replied:
здесь нужны ф-строки

NataliaHimera replied:
"""
Дмитрий, да, не учла равенство. Здесь еще 5 и 6 задание объединены, не так, как в задании на сайте GB, получилась путаница.
"""

revenue = float(input("Введите значение выручки фирмы, руб: "))
costs = float(input("Введите значение издержек фирмы, руб: "))
profit = revenue - costs

if profit > 0:
    print("Поздравляю с положительным финансовым результатом!))")
    print(f"Ваша прибыль составила {profit} руб.")
    rentability = profit / revenue * 100
    print(f"Рентабельность составила {rentability} %")
    employees = float(input("Сколько человек работает в вашей фирме?: "))
    profit_per_person = profit / employees
    print(f"Прибыль на одного сотрудника составила {profit_per_person} руб.")

elif profit < 0:
    print("К сожалению, ваша фирма терпит убытки((")
    print(f"Ваш убыток составил {profit} руб.")

else:
    print("Ваша фирма сработала в 0 - выручка равна издержкам")

DmitryChitalov commented on Урок 1. Практическое задание/task_6.py:
```diff
@@ -19,3 +19,35 @@
+while t < b:
+    d += 1
+    t = t * 1.1
+    print("Day " + str(d) + " Total: " + str(t))
```
выполнено

NataliaHimera committed:
Update task_1.py

Дмитрий, добрый день!
Высылаю ДЗ.

NataliaHimera committed:
Update task_2.py

NataliaHimera committed:
Update task_3.py

NataliaHimera committed:
Update task_4.py

NataliaHimera committed:
Update task_5.py

Сначала я хотела использовать здесь сортировку, что было бы проще и код короче, но потом подумала, что это будет не совсем правильно, так как в задании сказано, что элемент должен именно вставляться в определенное место. 
К сожалению, пока совсем нет времени на 6 задачку, а нужно уже браться за 3 урок.
